packages/block-library/src/navigation/constants.js

```diff
@@ -19,3 +19,14 @@ export const PRIORITIZED_INSERTER_BLOCKS = [
 	'core/navigation-link/page',
 	'core/navigation-link',
 ];
+
+export const SELECT_NAVIGATION_MENUS_ARGS = [
+	'postType',
+	'wp_navigation',
+	{
+		per_page: -1,
```

Comment:
Won't this need to be `100` to match with the PHP preloading? I think we have middleware on the _client_ side that doesn't allow unbounded queries (i.e. `-1`) but worth double checking.